ledger/participant-integration-api/src/main/scala/platform/store/interfaces/TransactionLogUpdate.scala

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.interfaces
+
+import java.time.Instant
+import com.daml.lf.value.{Value => LfValue}
+
+import com.daml.ledger.participant.state.v1.Offset
+import com.daml.lf.data.Ref.IdString
+import com.daml.lf.ledger.EventId
+import com.daml.platform.store.appendonlydao.events
+import com.daml.platform.store.appendonlydao.events.{ContractId, Identifier}
+import com.daml.platform.store.cache.MutableCacheBackedContractStore.EventSequentialId
+
+trait TransactionLogUpdate extends Product with Serializable
+
+object TransactionLogUpdate {
```

Comment:
I'd expect some commentary here, as to what the purpose of the classes within this `object` are. Otherwise, readers can only infer it from how it is currently being used, which seems like a recipe for keeping accidental dependencies intact; and adding new ones.

Reply:
Added docs